src/main/res/values/strings.xml

```diff
@@ -110,20 +110,20 @@
     <string name="settings">Settings</string>
     <string name="processing_FORMAT">Processing %1$s…</string>
     <string name="project_website">Project website</string>
-    <string name="promotes_non_free_network_services">Promotes non-free network services</string>
-    <string name="promotes_non_free_software">Promotes non-free software</string>
+    <string name="promotes_non_free_network_services">Promotes non-libre network services</string>
+    <string name="promotes_non_free_software">Promotes non-libre software</string>
     <string name="provided_by_FORMAT">Provided by %s</string>
     <string name="proxy">Proxy</string>
-    <string name="proxy_host">Proxy host</string>
-    <string name="proxy_port">Proxy port</string>
-    <string name="proxy_type">Proxy type</string>
-    <string name="recently_updated">Recently updated</string>
+    <string name="proxy_host">Host</string>
+    <string name="proxy_port">Port</string>
+    <string name="proxy_type">Type</string>
+    <string name="recently_updated">Recently synced</string>
     <string name="repositories">Repositories</string>
     <string name="repository">Repository</string>
-    <string name="repository_not_used_DESC">This repository has not been used yet. Turn it on to view the apps in it.</string>
-    <string name="repository_unsigned_DESC">Unsigned. Could not verify the app list. Be careful downloading apps from unsigned repositories.</string>
+    <string name="repository_not_used_DESC">This repository has not been used yet.\nConnect to it to view the apps in it.</string>
```

Comment:
I guess "enable" would be the right one here. Connect to it is too technical and turn it on is too simplistic

Reply:
Connect to it has some hope of being understood if the user doesn't know what a repo is.
Lets say there is confusion over whether something has to be done to the repo itself.
From "enable" it isn't clear that the repo is operated by someone else.